lib/process-services-cloud/src/lib/task/task-list/components/service-task-list-cloud.component.spec.ts

```diff
@@ -299,13 +282,9 @@ describe('ServiceTaskListCloudComponent', () => {
             expect(getServiceTaskByRequestSpy).toHaveBeenCalled();
         });
 
-        it('should reset pagination when resetPaginationValues is called', (done) => {
+        it('should reset pagination when resetPaginationValues is called', () => {
```

Comment:
The refactoring of this test doesn't take in account the fact that some calls were made, without the done the expectation won't work but this will give you a nice green passing test.

Reply:
:D i believe you but this might be different once we start running this unit tests after a while. if there's a subscription add the done please.